src/routes/Pages/Guides/Mage.js

```diff
@@ -118,15 +118,15 @@ const data = [
   {
     question:
       "Ok, I'm casting a lot of spells and running out of magic a lot now, is there a better way to get it back besides sleeping?",
-    answer: `From levels 1 to 3 you can drink from the Cabeiri fountain. After that you'll probably want to start buying mana orbs in Whitestorm for 35 platinum each. The shop is to the west, in the magic district. These orbs heal your magic constantly and much faster than sleep. You can have up to 3 draining at a time, with 3 in reserve. If you buy more than 6, they will start to meld together randomly. There are some other ways to recover magic, as well as other orb shops, but you'll have to find those for yourself.`,
+    answer: `From levels 1 to 3 you can drink from the Cabeiri fountain. After that you'll probably want to start buying mana orbs in Whitestorm for 35 platinum each. The shop is to the west, in the magic district. These orbs heal your magic constantly and much faster than sleep. You can have up to 3 draining at a time, with extras reserve. You can change the speed at which they drain with "drain <speed>" (quickly, normally, or slowly). There are some other ways to recover magic, as well as other orb shops, but you'll have to find those for yourself.`,
```

Comment:
Since we're updating this section, we should probably update the blurb about the fountain, too. It was changed a few years back to work from levels 1 to 5.

Reply:
maybe there should be a blurb about buying newbie orbs in cab at 3W, 2N, 1E as well